packages/dev/addons/src/lottie/lottiePlayer.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * Configuration options for the Lottie animation player.
+ */
+export type AnimationConfiguration = {
+    /**
+     * Whether the animation should play on a loop or not
+     */
+    loopAnimation?: boolean;
+    /**
+     * Size of the sprite atlas texture.
+     * Default is 2048.
+     */
+    spriteAtlasSize?: number;
+    /**
+     * Gap size around sprites in the atlas.
+     * Default is 5.
+     */
+    gapSize?: number;
+    /**
+     * Maximum number of sprites the renderer can handle at once.
+     * Default is 64.
+     */
+    spritesCapacity?: number;
+    /**
+     * Background color for the animation canvas.
+     * Default is white with full opacity.
+     */
+    backgroundColor?: { r: number; g: number; b: number; a: number };
+    /**
+     * Minimum scale factor to prevent too small sprites.
+     * Default is 5.
+     */
+    scaleMultiplier?: number;
+    /**
+     * Scale factor for the rendering.
+     * Default is 1.
+     */
+    devicePixelRatio?: number;
+    /**
+     * Number of steps to sample cubic bezier easing functions for animations.
+     * Default is 4.
+     */
+    easingSteps?: number;
+    /**
+     * Whether to ignore opacity animations for performance.
+     * Default is true.
+     */
+    ignoreOpacityAnimations?: boolean;
+    /**
+     * Whether to support device lost events for WebGL contexts.
+     * Default is false.
+     */
+    supportDeviceLost?: boolean;
+};
+
+/**
+ * LottiePlayer is a class that allows you to play Lottie animations using Babylon.js.
+ * It plays the animations in a worker thread using OffscreenCanvas.
+ * Once instance of this class can only be used to play a single animation. If you want to play multiple animations, create a new instance for each animation.
+ */
+export class LottiePlayer {
+    private _playing: boolean;
+    private _disposed: boolean;
+    private _worker: Worker | null;
+    private _canvas: HTMLCanvasElement | null;
+
+    /**
+     * Creates a new instance of the LottiePlayer.
+     */
+    public constructor() {
+        this._playing = false;
+        this._disposed = false;
+        this._worker = null;
+        this._canvas = null;
+    }

```

Comment:
Maybe init inline and use Nullable?
```suggestion
    private _playing = false;
    private _disposed = false;
    private _worker: Nullable<Worker> = null;
    private _canvas: Nullable<HTMLCanvasElement> = null;
```

Reply:
I'll update the code in general to init inline and use Nullable.